flow-typed/npm/@meetup/swarm-components_vx.x.x.js

```diff
@@ -0,0 +1,137 @@
+// flow-typed signature: d0c29d5fab5a31c6bb97b6f515e1a675
```

Comment:
Hmm what are all these flow-typed files in the diff?

Reply:
They are generated stubs of the packages that lacked `flow-typed` definitions

Reply:
It's helpful indicate the generator source when referring to generated files - e.g. "I ran `flow-typed` in order to import the latest flow type definitions for package dependencies - these are the stubs it created for packages that do not have flow-typed libdefs https://github.com/flowtype/flow-typed/

Reply:
also, just in case it's helpful - I have sometimes run `flow-typed` on its own branch because it make diffs very difficult to read. It's helpful to keep all the irrelevant updates isolated from feature branches